plop-templates/package/babel.config.js

```diff
@@ -0,0 +1 @@
+module.exports = require('../../babel.config')
```

Comment:
nit: I have to look and see what this was inheriting from. I guessed it was the root babel config. Not a big deal but a small comment stating that being the